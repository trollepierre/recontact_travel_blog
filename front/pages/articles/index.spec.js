import ArticlesIndex from '@/pages/articles/index'

describe('Page | index.vue', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallowMount(ArticlesIndex)

    // Then
    expect(wrapper).toMatchSnapshot()
  })
})
