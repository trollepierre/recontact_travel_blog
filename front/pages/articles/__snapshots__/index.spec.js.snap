// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page | index.vue should match snapshot 1`] = `
<div>
  <app-header-stub></app-header-stub>
  <main>
    <article-list-stub articlesnumberlimit="0"></article-list-stub>
  </main>
</div>
`;
